### DISCUSSION TITLE
Would it be possible to make a version based on the Evergreen webview2 runtime?

### DISCUSSION THREAD

**Discussion by femi-at-axis** [r0x0r/pywebview#1139] [Ideas]
There is a known issue that the fixed runtime of webview2 cannot run from a network location, which means that pywebview apps crash when you run from there.

https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#known-issues-for-fixed-version

Could pywebview use this or possibly the builtin webview2 runtime, since it seems to be preinstalled now on windows 10
https://www.bleepingcomputer.com/news/microsoft/microsoft-is-auto-installing-the-windows-10-webview2-runtime/

**Comment by firai** [comment]
I understand that Pywebview uses the Evergreen version if it is installed. Note that the Pywebview installer doesn't install Webview2 by itself (see https://pywebview.flowrl.com/guide/installation.html#windows) and that the loader dll's don't include the Webview2 runtime. Maybe your issue is related to https://github.com/MicrosoftEdge/WebView2Feedback/issues/3465?

**Comment by r0x0r** [comment]
@firai is correct. `pywebview` does not distribute webview2 runtime, but uses what user has installed.
